digits/model/tasks/train.py

```diff
@@ -58,6 +58,9 @@ def __init__(self, job, dataset, train_epochs, snapshot_interval, learning_rate,
         self.shuffle = kwargs.pop('shuffle', None)
         self.network = kwargs.pop('network', None)
         self.framework_id = kwargs.pop('framework_id', None)
+        self.mean_path = kwargs.pop('mean_path', None)
```

Comment:
I think it makes more sense to have `mean_path` be a property of the model job rather than a property of the train task. The mean file is used during inference too so it's not just required during training.